cdk/emr-serverless-with-sfn/stacks/emr_serverless.py

```diff
@@ -0,0 +1,619 @@
+import aws_cdk as cdk
+from aws_cdk import CfnOutput, Duration, Stack
+from aws_cdk import aws_cloudwatch as cw
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_emrserverless as emrs
+from constructs import Construct
+
+
+class EMRServerlessStack(Stack):
+    serverless_app: emrs.CfnApplication
+
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        vpc: ec2.IVpc,
+        namespace: str,
+        **kwargs,
+    ) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        # Create an EMR 6.6.0 Spark application in a VPC with pre-initialized capacity
+        self.serverless_app = emrs.CfnApplication(
+            self,
+            f"spark_app_{namespace}",
+            release_label="emr-6.11.0",
+            type="SPARK",
+            name=f"spark-app-{namespace}",
+            network_configuration=emrs.CfnApplication.NetworkConfigurationProperty(
+                subnet_ids=vpc.select_subnets().subnet_ids,
+                security_group_ids=[self.create_security_group(vpc).security_group_id],
+            ),
+            initial_capacity=[
+                emrs.CfnApplication.InitialCapacityConfigKeyValuePairProperty(
+                    key="Driver",
+                    value=emrs.CfnApplication.InitialCapacityConfigProperty(
+                        worker_count=2,
+                        worker_configuration=emrs.CfnApplication.WorkerConfigurationProperty(
+                            cpu="4vCPU", memory="16gb"
+                        ),
+                    ),
+                ),
+                emrs.CfnApplication.InitialCapacityConfigKeyValuePairProperty(
+                    key="Executor",
+                    value=emrs.CfnApplication.InitialCapacityConfigProperty(
+                        worker_count=10,
+                        worker_configuration=emrs.CfnApplication.WorkerConfigurationProperty(
+                            cpu="4vCPU", memory="16gb"
+                        ),
+                    ),
+                ),
+            ],
+            auto_stop_configuration=emrs.CfnApplication.AutoStopConfigurationProperty(
+                enabled=True, idle_timeout_minutes=100
```

Comment:
Maybe we should decrease this for the sample? 100 minutes seems quite long.
```suggestion
                enabled=True, idle_timeout_minutes=15
```